DigitalLearningSolutions.Data/Models/User/EditAccountDetailsData.cs

```diff
@@ -20,10 +20,10 @@ public class EditAccountDetailsData : AccountDetailsData
             ProfileImage = profileImage;
         }
 
-        public int UserId { get; set; }
-        public byte[]? ProfileImage { get; set; }
-        public int JobGroupId { get; set; }
-        public string? ProfessionalRegistrationNumber { get; set; }
-        public bool HasBeenPromptedForPrn { get; set; }
+        public int UserId { get; }
+        public byte[]? ProfileImage { get; }
+        public int JobGroupId { get; }
+        public string? ProfessionalRegistrationNumber { get; }
+        public bool HasBeenPromptedForPrn { get; }

```

Comment:
Convert to `public readonly ...` as above

Reply:
(We've decided not to do this)